libs/island-ui/core/src/lib/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { PropsWithChildren } from 'react'
+import { Button } from '../Button/Button'
+import { Box } from '../Box/Box'
+import { ModalBase } from '../ModalBase/ModalBase'
+import cn from 'classnames'
+import * as styles from './Drawer.css'
+
+interface DrawerProps {
+  /**
+   * Explain what this drawer is for
+   */
+  ariaLabel: string
+  /**
+   * Unique ID for accessibility purposes
+   */
+  baseId: string
+  /**
+   * Element that opens the drawer.
+   * It will be forwarded neccessery props for a11y and event handling.
+   */
+  disclosure: React.ReactElement
+  /**
+   * Show immediately without clicking the disclosure button
+   */
+  initialVisibility?: boolean | undefined
+  /**
+   * Position of the drawer
+   */
+  position?: 'right' | 'left'
+}
+
+export const Drawer = ({
```

Comment:
What does this do? Does it feature deploy somewhere? Figma link?

Reply:
It's basically a Modal on the side if the screen

See design here (when clicking on a table row): https://www.figma.com/proto/Nz4FjxCAhpaHqjyK8mrDEy/Ums%C3%B3knarkerfi%C3%B0?node-id=1995-44442&starting-point-node-id=1995%3A44442

Thought it would make sense to have as a shared UI component as it will probably be needed elsewhere in the future, we will get our designer to add this as a proper UI component in Figma